lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/discussion/DiscussionThreadBundle.json

```diff
@@ -27,6 +31,13 @@
 			"format":"date-time",
 			"description":"The timestamp when this Thread was last modified"
 		},
+		"etag": {
+			"type": "string"
```

Comment:
description: Synapse employs an Optimistic Concurrency Control (OCC) scheme to handle concurrent updates. Since the E-Tag changes every time an entity is updated it is used to detect when a client's current representation of an entity is out-of-date.